dev/AppLifecycle/FileActivatedEventArgs.h

```diff
@@ -44,11 +44,11 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
             }
         }
 
-        static IActivatedEventArgs Deserialize(winrt::Windows::Foundation::Uri const& uri)
+        static winrt::Windows::Foundation::IInspectable Deserialize(winrt::Windows::Foundation::Uri const& uri)
         {
             auto query = uri.QueryParsed();
-            std::wstring verb = query.GetFirstValueByName(L"Verb").c_str();
-            std::wstring file = query.GetFirstValueByName(L"File").c_str();
+            auto verb = std::wstring(query.GetFirstValueByName(L"Verb"));
+            auto file = std::wstring(query.GetFirstValueByName(L"File"));
```

Comment:
Nit: Should these be passed around as winrt::hstring? They come out of the decoder as winrt::hstring, so we're up to like three copies here:

1. From hstring -> wstring
2. Passing wstring local to value-typed parameter
3. Moving from value-typed parameter to member

If this was hstring everywhere that'd be three reference bumps and no copies.

Reply:
Adding this comment to this open issue in AppLifeycle: #926